### PR TITLE
Added reuse as a test env in travis.

### PR DESCRIPTION
**Pull request by amrohassaan**
Since old PR seems closed, pushing new one with required changes. Added reuse as a test job in travis.yml

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ env:
   - SCRIPT="./dev-docker ./manage.py test"
   - SCRIPT="./scripts/test-docker"
   - SCRIPT="./manage.py compilemessages" INSTALLPIP=yes
+  - SCRIPT="docker run -v $(PWD):/src --workdir /src fsfe/reuse reuse lint"
```
Comment (chaws) [inline .travis.yml]: I think you mean `squad` instead of `src`, which is where reuse will scan our source files.

Comment (amrohassaan) [reply .travis.yml]: Ack, thanks @chaws

File: .travis.yml [hunk 2]
```diff
@@ -20,6 +20,7 @@ env:
   - SCRIPT="./dev-docker ./manage.py test"
   - SCRIPT="./scripts/test-docker"
   - SCRIPT="./manage.py compilemessages" INSTALLPIP=yes
+  - SCRIPT="docker run -v $(pwd):/squad --workdir /squad/squad fsfe/reuse reuse lint"
```
Comment (terceiro) [inline .travis.yml]: why `/squad/squad`? with this you are running from inside the `squad` subdirectory, instead of from the root of the repository.

Comment (amrohassaan) [reply .travis.yml]: I was just having a chat with @chaws right now about it ..we thought /squad is where main dir under the docker container and then /squad/squad is where the source files are. We were having doubts and not sure as well..so It should be 
docker run -v $(pwd):/squad --workdir /squad/ fsfe/reuse reuse lint?

Comment (chaws) [reply .travis.yml]: @terceiro should we run reuse on non-source files as well?

Comment (mwasilew) [reply .travis.yml]: It should be run in top level dir. We have scripts and tests dirs which are also part of SQUAD.

Comment (chaws) [reply .travis.yml]: ack

Comment (amrohassaan) [reply .travis.yml]: @mwasilew exactly yes.

**Comment by terceiro** [comment]
the commit message mentions a fix, but what is being fixed will never appear in the repository. so use a commit message that just says what the patch does in terms of the current contents of the repository, not in terms of previous versions of the patch,

**Comment by amrohassaan** [comment]
@terceiro ack. Amended comment and repushed.

**Pull request merged**